numpy/core/tests/test_umath.py

```diff
@@ -2460,6 +2461,42 @@ def test_builtin_long(self):
         assert_equal(np.gcd(2**100, 3**100), 1)
 
 
+class TestRoundingFunctions(object):
+
+    def test_object_direct(self):
+        """ test direct implementation of these magic methods """
```

Comment:
I think pytest is OK with normal docstrings, they were a problem for nose. @rgommers Anyone know?

Reply:
I've not seen any problem with them. I know nose produced confusing output with them, but I think pytest is more useful.

Reply:
IIRC, the nose problems were mostly when using the `yield` testing option, which pytest doesn't support.